script/setup_local_certs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# TODO what about non-Mac on Zscaler?
+mkdir -p .certs
+
+# Eject the usual CA certificates for the local host
+security export -t certs -f pemseq -k /System/Library/Keychains/SystemRootCertificates.keychain -o .certs/ca-certificates.crt
+
+# Append the Zscaler CA certificate to the bundle
+security find-certificate -p -c "Zscaler" /Library/Keychains/System.keychain >> .certs/ca-certificates.crt
```

Comment:
The certificate filtering and addition should be optional and user-supplied - not hardcoded into the tool.